Zend/zend_fibers.c

```diff
@@ -734,17 +691,13 @@ ZEND_METHOD(Fiber, getReturn)
 
 ZEND_METHOD(Fiber, this)
 {
-	zend_fiber *fiber;
-
 	ZEND_PARSE_PARAMETERS_NONE();
 
-	fiber = EG(current_fiber);
-
-	if (!fiber) {
+	if (EG(current_fiber) == EG(main_fiber)) {
 		RETURN_NULL();
 	}
 
-	RETURN_OBJ_COPY(&fiber->std);
+	RETURN_OBJ_COPY(&zend_fiber_from_context(EG(current_fiber))->std);
```

Comment:
This is easier to read as (zend_object*) zend_fiber_from_context(EG(current_fiber))

Reply:
I don’t think it’s a good idea to cast pointers, and we hope that we can drop the final in the future, so std may be the last one.